services/impl/src/main/java/org/exoplatform/automatic/translation/rest/AutomaticTranslationRestService.java

```diff
@@ -53,57 +54,31 @@ public AutomaticTranslationRestService(AutomaticTranslationService automaticTran
   @Path("/configuration")
   @RolesAllowed("administrators")
   @Produces(MediaType.APPLICATION_JSON)
-  @ApiOperation(
-      value = "Gets Automatic Translation configuration",
-      httpMethod = "GET",
-      response = Response.class,
-      produces = MediaType.APPLICATION_JSON,
-      notes = "This returns the actual configuration for automatic translation"
-  )
-  @ApiResponses(
-      value = {
-          @ApiResponse(code = 200, message = "Request fulfilled")
-      }
-  )
+  @ApiOperation(value = "Gets Automatic Translation configuration", httpMethod = "GET", response = Response.class, produces = MediaType.APPLICATION_JSON, notes = "This returns the actual configuration for automatic translation")
+  @ApiResponses(value = { @ApiResponse(code = 200, message = "Request fulfilled") })
```

Comment:
We can have an HTTP 500 response when an error happens + 403 when the user is not allowed to get the configuration (not member of /platform/administrators)

Reply:
done